perception/object_merger/src/object_association_merger/node.cpp

```diff
@@ -34,9 +34,9 @@ namespace
 bool isUnknownObjectOverlapped(
   const autoware_auto_perception_msgs::msg::DetectedObject & unknown_object,
   const autoware_auto_perception_msgs::msg::DetectedObject & known_object,
-  const double precision_threshold, const double recall_threshold)
+  const double precision_threshold, const double recall_threshold, double distance_threshold)
 {
-  constexpr double sq_distance_threshold = std::pow(5.0, 2.0);
+  const double sq_distance_threshold = std::pow(distance_threshold, 2.0);
```

Comment:
I want to separate distance_threshold by object class.
For bus, a large value is fine, but for pedestrian, 2[m] is enough.

Reply:
I changed distance_threshold to distance_threshold_list, and it enables to separate distance_threshold by object class.